src/standalone/Vegetation/Canopy.jl

```diff
@@ -147,8 +148,7 @@ end
 
 Creates a PlantHydraulicsModel on the provided domain, using default parameters.
 
-The required argument `forcing` should be a NamedTuple with the following field:
-- `LAI`: a function or ClimaUtilities TimeVaryingInput for leaf area index
+The required argument `LAI` should be a function or ClimaUtilities TimeVaryingInput for leaf area index.
```

Comment:
This is minor and doesnt have to be in this PR, because I think we do this in a lot of places, but the time varying input object can also encompass an analytic function, so as long as the inputs to the function are (t), we can remove the `Function` type